src/daemon/dlt-daemon.c

```diff
@@ -3246,6 +3246,9 @@ int dlt_daemon_close_socket(int sock, DltDaemon *daemon, DltDaemonLocal *daemon_
         // In the offline trace AND in the socket stream.
         if(daemon_local->flags.yvalue[0] == 0)
             dlt_daemon_change_state(daemon,DLT_DAEMON_STATE_BUFFER);
+
+        //send connection info to buffer, if there are not any clients available
+        dlt_daemon_control_message_connection_info(DLT_DAEMON_SEND_TO_ALL,daemon,daemon_local,DLT_CONNECTION_STATUS_DISCONNECTED,"",verbose);
     }
```

Comment:
when there is no client, whom this control message is sent? i didnt understand this change. Can you please explain what is the use case which triggered you this change?

Reply:
Still i couldn't understand. BTW, Can you please explain what is the use case which triggered you this change?

Reply:
Originally, the connection info control message was sent always, when there was a client disconnected. But I suppose more reliable information is, to have info when the last client is disconnected (no more clients are available). I suppose this was the main use case (debugging purposes).